management/src/test/java/org/ehcache/docs/ManagementTest.java

```diff
@@ -51,25 +55,26 @@ public void usingManagementRegistry() throws Exception {
         .withResourcePools(ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10, EntryUnit.ENTRIES).build())
         .buildConfig(Long.class, String.class);
 
-    ManagementRegistry managementRegistry = new DefaultManagementRegistry(); // <1>
+    DefaultManagementRegistryConfiguration registryConfiguration = new DefaultManagementRegistryConfiguration().setCacheManagerAlias("myCacheManager"); // <1>
+    ManagementRegistry managementRegistry = new DefaultManagementRegistry(registryConfiguration); // <2>
```

Comment:
This is not needed as it would be handled automatically by the service locator. Our samples should promote recommended usage.